### PR TITLE
Update ember-data.md

### PR DESCRIPTION
**Pull request by aureliosaraiva**
Parte da tradução de tutorial
Issue: https://github.com/ember-brasil/website/issues/3

### CODE REVIEW

File: source/guides/tutorial/ember-data.md [hunk 1]
```diff
@@ -1,27 +1,24 @@
-Currently, our app is using hard-coded data for our rental listings, defined in the `rentals` route handler.
-As our application grows, we will want to persist our rental data on a server, and make it easier to do advanced operations on the data, such as querying.
+Atualmente, nossa aplicação está usando dados fake em nossa listagem de imoveis para alugar, que estão definidas no manipulador de route `rental`.
+À medida que nossa aplicação cresce, precisaremos persistir nossos imóveis em um servidor remoto para facilitar operações de consulta, criação e edição desses dados.
 
-Ember comes with a data management library called [Ember Data](https://github.com/emberjs/data) to help deal with persistent application data.
+O Ember vem com uma biblioteca para gerenciamento de dados chamada [Ember Data](https://github.com/emberjs/data), ela nos ajuda a persistir e consultar dados de uma servidor remoto.
```
Comment (danilovaz) [inline source/guides/tutorial/ember-data.md]: Fix typo: `de uma servidor remoto` p/ `de um servidor remoto`

File: source/guides/tutorial/ember-data.md [hunk 2]
```diff
@@ -1,27 +1,24 @@
...
 
-Ember Data requires you to define the structure of the data you wish to provide to your application by extending [`DS.Model`](http://emberjs.com/api/data/classes/DS.Model.html).
+O Ember Data precisa que especiquemos a estrutura dos dados que vamos utilizar em nossa aplicação, estendendo [`DS.Model`](http://emberjs.com/api/data/classes/DS.Model.html).
```
Comment (danilovaz) [inline source/guides/tutorial/ember-data.md]: Fix typo: `precisa que especiquemos` p / `precisa que especifiquemos`

File: source/guides/tutorial/ember-data.md [hunk 3]
```diff
@@ -1,27 +1,24 @@
...
 
-You can generate an Ember Data Model using Ember CLI.
-We'll call our model `rental` and generate it as follows:
+Vamos usar o Ember ClI para gerar nosso Ember Data Model com o nome `rental`:
```
Comment (danilovaz) [inline source/guides/tutorial/ember-data.md]: Fix typo: `Ember ClI` p/ `Ember CLI`

File: source/guides/tutorial/ember-data.md [hunk 4]
```diff
@@ -1,27 +1,24 @@
...
 
 ```shell
 ember g model rental
 ```
-
-This results in the creation of a model file and a test file:
+Será criado dois arquivos, um com nosso model e outro com nosso teste.
```
Comment (danilovaz) [inline source/guides/tutorial/ember-data.md]: Fix: `Será criado dois arquivos` p/ `Serão criados dois arquivos`

File: source/guides/tutorial/ember-data.md [hunk 5]
```diff
@@ -98,13 +95,11 @@ export default Ember.Route.extend({
 });
 ```
 
-When we call `findAll`, Ember Data will attempt to fetch rentals from `/api/rentals`.
-If you recall, in the section titled [Installing Addons](../installing-addons/) we set up an adapter to route data requests through `/api`.
+Quando chamamos `findAll`, o Ember Data tentará buscar os imoveis em `/api/rentals`.
```
Comment (danilovaz) [inline source/guides/tutorial/ember-data.md]: Fix typo: `imoveis` p/ `imóveis`

File: source/guides/tutorial/ember-data.md [hunk 6]
```diff
@@ -98,13 +95,11 @@ export default Ember.Route.extend({
...
-You can read more about Ember Data in the [Models section](../../models/).
+Você pode ler mais sobre Ember Data na seção [Model](../../models/).
 
-Since we have already set up Ember Mirage in our development environment, Mirage will return the data we requested without actually making a network request.
+Uma vez que já criamos o Ember Mirage no nosso ambiente de desenvolvimento, o Mirage retornará os dados que solicitamos, sem fazer requisições na rede.
 
-When we deploy our app to a production server,
-we will likely want to replace Mirage with a remote server for Ember Data to communicate with for storing and retrieving persisted data.
-A remote server will allow for data to be shared and updated across users.
+Quando publicarmos nassa aplicação em um servidor de produção, iremos substituir o Mirage por um servidor remoto, para que Ember Data possa manipular dados reais.
```
Comment (danilovaz) [inline source/guides/tutorial/ember-data.md]: Fix: `Quando publicarmos nassa` p/ `Quando publicarmos nossa`

**Comment by aureliosaraiva** [comment]
6 words might be misspelled, please check them: api, attr, findAll, js, recêm, toc

**Comment by aureliosaraiva** [comment]
4 words might be misspelled, please check them: attr, findAll, recêm, toc

**Comment by aureliosaraiva** [comment]
5 words might be misspelled, please check them: attr, findAll, findRecord, hbs, recêm

**Comment by aureliosaraiva** [comment]
1 words might be misspelled, please check them: hbs

**Comment by aureliosaraiva** [comment]
1 words might be misspelled, please check them: hbs

**Comment by aureliosaraiva** [comment]
No spelling errors, congratulations!

**Comment by aureliosaraiva** [comment]
No spelling errors, congratulations!

**Pull request merged**